ieeeuoft/newhacks-site-2022
# copied over basic styling from newhacks 2021

## Overview

- Resolves #NH-4


## Unit Tests Created

- none


## Steps to QA

- after you pull the branch you have to run `yarn run scss` and `python manage.py collectstatic` to compile the stying
- stop your hackathon-template docker containers and use the newhacks-development folder to start the newahcks containers
- go to localhost:8000 and make sure the styling is correct for all pages
